web/src/main/scala/net/wiringbits/components/widgets/Footer.scala

```diff
@@ -69,7 +69,7 @@ import slinky.core.facade.Fragment
     val copyright = Container(
       margin = Container.EdgeInsets.vertical(margin),
       alignItems = Container.Alignment.center,
-      child = mui.Typography(s"${AppStrings.appName} 2021").color(Color.inherit)
+      child = mui.Typography(s"${texts.appName} 2021").color(Color.inherit)
```

Comment:
Actually, you can do `texts.appNameCopyright` which uses the current year instead of a hardcoded one